File: 6. Arreglos/c++/main.cpp
#include <iostream>
using namespace std;

int main()
{
    int x=2, y=2;    
    int **matriz;
    int vector[3]={1,2,3};
    for(int i=0;i<3;i++)
        cout<<vector[i]<<"\n";
    matriz = (int **)malloc (y*sizeof(int *));    
    for (int i=0;i<y;i++)
    matriz[i] = (int *) malloc (x*sizeof(int));
    int cont=0;
    for(int i=0;i<y;i++)
    {
        for(int j=0;j<x; j++)
        {
            matriz[i][j]=cont;
            cout<<matriz[i][j];
            cont++;
        }
        cout<<"\n";
    }
}

File: 6. Arreglos/js/main.js
var items = [
    [1, 2],
    [3, 4],
    [5, 6]
  ];
  print(items[0][0]); // 1
  print(items);
  
  var x = new Array(3);
  
  for (var i = 0; i < x.length; i++) {
    x[i] = new Array(3);
  }
  
  var cont=0;
  for(var i=0; i<x.length;i++)
      for(var j=0; j <x[i].length;j++)
      {
          x[i][j]=cont;
          print(x[i][j]);
          cont++;
      }
  print(x);